cli/documentation/docs/01_installation.md

```diff
@@ -0,0 +1,42 @@
+# Installation
+
+## From release
+
+### 1. Download
+
+Go to https://github.com/iotaledger/cli-wallet/releases and download the latest release binary for your platform.
```

Comment:
For now probably still makes sense, but for the next release not anymore, should we already change?

Reply:
Yeah, when it's merged we can do a new release, it's been a long time anyway